unit_tests/test_config.py

```diff
@@ -350,6 +350,13 @@ def test_15_new_scylla_repo(self):
 
         conf = SCTConfiguration()
         conf.verify_configuration()
+
+        original_get_branch_version = sdcm.sct_config.get_branch_version
+        sdcm.sct_config.get_branch_version = lambda x: '2019.1.1'
+
+        conf._get_target_upgrade_version()
+
+        sdcm.sct_config.get_branch_version = original_get_branch_version
```

Comment:
De-facto you mock stuff here but do it in a strange way. Why "mock" library is not used?

Please, reuse "mock" library: https://docs.python.org/3.8/library/unittest.mock.html